drivers/storage/portworx/component/telemetry.go

```diff
@@ -581,6 +588,13 @@ func (t *telemetry) createCCMGoConfigMapRegister(
 	if err != nil {
 		return err
 	}
+
+	// Check PX version to update the config for "container orchestrator"
+	if pxutil.IsCOSupported(cluster) {
+		config = certStoreTypeRegx.ReplaceAllString(config,
+			"certStoreType: \"kvstore\"\nkvStore:\n  serverAddress: \"127.0.0.1:"+strconv.Itoa(GetPxCOListeningPort(cluster))+"\"")
```

Comment:
not sure if i'm readying this right. But should we add a new line at the end too?

Reply:
The configmap by defaults contains 
```
cloud:
      auth:
        version: "1.0"
    productName: "portworx"
    certStoreType: "k8s"
    k8s:
      certSecretName: "pure-telemetry-certs"
      certSecretNamespace: kube-test
``` 
after the replace code it already has a newline
```
 cloud:
      auth:
        version: "1.0"
    productName: "portworx"
    certStoreType: "kvstore"
    kvStore:
      serverAddress: "127.0.0.1:9030"
    k8s:
      certSecretName: "pure-telemetry-certs"
      certSecretNamespace: portworx
```